### DISCUSSION TITLE
Analyzer doesn't emit RMG020 if the target class misses a property

### DISCUSSION THREAD

**Discussion by StasPerekrestov** [riok/mapperly#344] [Q&A]
Hello,

I'm looking into the library and trying to understand
how to enable https://mapperly.riok.app/docs/configuration/analyzer-diagnostics messages.
I have two almost identical classes, except the destination class misses one property.
What I need help figuring out is how to get the RMG020 message.
Is there any toggle to switch on to make the diagnostic message emitted?

Thank you.

**Comment by latonz** [comment]
All diagnostics are reported based on their default severity. `RMG020` has a default severity of Info and is reported as Info unless configured otherwise.
Do you have a editorconfig rule for RMG020? Could you provide the classes and the mapper involved as a minimal reproducible?

**Comment by StasPerekrestov** [comment]
Hello Lars,
Thank you for giving me a hand with this.
It looks like I found the issue.
My editor config file was missing `[*.cs]`

```.editorconfig
dotnet_diagnostic.RMG012.severity = error # Unmapped target property
dotnet_diagnostic.RMG020.severity = error # Unmapped source property
```
vs
```.editorconfig
[*.cs]
dotnet_diagnostic.RMG012.severity = error # Unmapped target property
dotnet_diagnostic.RMG020.severity = error # Unmapped source property
```
After I added `[*.cs]` to the file, everything started working as expected.

Thank you.
